integrations/docker/images/vscode/chip-build-vscode/Dockerfile

```diff
@@ -144,6 +143,10 @@ ENV ZEPHYR_NXP_BASE=/opt/nxp-zephyr/zephyrproject/zephyr
 ENV ZEPHYR_NXP_SDK_INSTALL_DIR=/opt/nxp-zephyr/zephyr-sdk-0.16.5
 ENV NXP_UPDATE_SDK_SCRIPT_DOCKER=/opt/nxp/nxp_matter_support/scripts/update_nxp_sdk.py
 
+# Places bootstrap files there instead of the default one which is `.environment`.
+# NOTE: This directory is NOT persistent.
+ENV PW_ENVIRONMENT_ROOT=/home/vscode/pigweed/env
```

Comment:
how about keeping the previous placement and only adding the comment? That way I seem to be more ensured that docker will make these files identical and I can justify no version increase

Reply:
Overall it looks to me like env variables were alpha-sorted. The new place seems a bit off from that perspective.

Reply:
> env variables were alpha-sorted

More or less, but in general these envs were not sorted. The case with `PW_ENVIRONMENT_ROOT` is that it does not affect the container itself, but its a sane default for a tool which is not baked into the image. The rest of the env variables if for SDKs baked into the image (`/opt/...`). There is also `PATH` variable which might get "lost" if all these envs were alpha-sorted, and I would propose to keep it separately as well.

And of course maybe we could sort all these envs now?

Reply:
These envs (in my opinion) should, and somewhat are, grouped by meaning before being sorted. `PW_ENVIRONMENT_ROOT` is not an SDK path so it might not be the best idea to mix it with SDK paths, especially considering it's importance. If a major restructure of this Dockerfile is wanted it probably should be done in other PR.